python-heasoft-osa/Dockerfile

```diff
@@ -0,0 +1,59 @@
+# For finding latest versions of the base image see
+# https://github.com/SwissDataScienceCenter/renkulab-docker
+#ARG RENKU_BASE_IMAGE=renku/renkulab-py:3.7-0.7.3
+#FROM ${RENKU_BASE_IMAGE}
+FROM integralsw/osa-python:11.1-16-g88c002b7-20210507-170349-refcat-43.0-heasoft-6.29-python-3.8.2
```

Comment:
is there some semver-compatible version of this tag? If yes, then we can have dependabot update the images automatically when new ones get released